src/utils/github.js

```diff
@@ -15,51 +15,112 @@ export async function createRepo (githubOrg, githubRepo, templateOrg, templateRe
   // Check if the repository already exists
   const cmdResult = await runCommand(`gh api repos/${githubOrg}/${githubRepo}`)
   if (cmdResult?.stdout) { // if not exist, will throw and return 404.
-    // // If the repository exists, ask the user if they want to overwrite its content
-    // const overwrite = await promptConfirm(
-    //     `The repository ${githubOrg}/${githubRepo} already exists. Do you want to overwrite its content?`
-    // )
-    // // If the user wants to overwrite the repo, use "update" instead of "create"
-    // if (overwrite) {
-    //   // TODO: implement overwrite logic
-    // } else {
-    //   // If the user doesn't want to overwrite the repo, exit the program
-    //   console.error(`❌ Exiting. Cannot create repository that already exists: ${githubOrg}/${githubRepo}`)
-    //   process.exit(1)
-    // }
     console.error(`❌ Skipping. Cannot create repository that already exists: ${githubOrg}/${githubRepo}`)
   } else {
     // If the repository does not exist, proceed with "create"
     await runCommand(`gh repo create ${githubOrg}/${githubRepo} --template ${templateOrg}/${templateRepo} --public`)
+    // without timeout the commits are all out of order for some reason, and "initial commit" is the last, so fstab and other things are wiped although the appear in commit history
+    // TODO figure out how to create the commits in proper order without having a timeout or delay.
+    await new Promise(resolve => {
+      setTimeout(() => resolve(), 5000)
+    })
     console.log(`✅ Created code repository at https://github.com/${githubOrg}/${githubRepo} from template ${templateOrg}/${templateRepo}`)
     await modifyFstab(githubOrg, githubRepo, templateRepo)
     await modifySidekickConfig(githubOrg, githubRepo)
+
+    // aem-boilerplate-commerce is on config service, so for scaffolded repos we need to create a local config.json
+    if (templateRepo === 'aem-boilerplate-commerce') {
+      await createLocalCommerceConfig(githubOrg, githubRepo, templateOrg, templateRepo)
+    }
```

Comment:
Is this a snowflake implementation while something is being worked on the `aem` side?

Reply:
This is going to be necessary for the other repos if they use config service too.